index.js

```diff
@@ -0,0 +1,15 @@
+/*eslint-disable*/
+import BooksdataBase from './Modules/library.js';
+import pageNav from './Modules/navigation.js';
+
+const date = document.getElementById('date');
+
+window.setInterval(() => {
+  // eslint-disable-next-line no-undef
```

Comment:
- [ ] You are not supposed to change the Linters rule, instead fix the linters error. Also, make sure to check through all your files and fix these linters issues instead of disabling them. Linters are tools that analyze your codes and flag programming errors hence fix these issues.